resources/js/tryitout.js

```diff
@@ -61,9 +72,7 @@ function makeAPICall(method, path, body, query, headers) {
     const url = new URL(window.baseUrl + '/' + path.replace(/^\//, ''));
     Object.keys(query).forEach(key => {
         if (typeof query[key] === 'object') {
-            Object.keys(query[key]).forEach(objKey => {
-                url.searchParams.append(`${key}[${objKey}]`, query[key][objKey]);
-            });
+            recursiveAddSearchParams(url, key, null, query[key]);
```

Comment:
We don't need the outer `Object.keys` loop anymore. We can simply call the recursive function right after the `const url` line. It should work, since the recursive function also handles non-object values.